crates/timers/src/future.rs

```diff
@@ -1,8 +1,11 @@
 //! `Future`- and `Stream`-backed timers APIs.
 
+use std::task::{Poll, Context};
+use std::pin::Pin;
+use std::future::Future;
 use super::sys::*;
```

Comment:
Could you move the super import to the top? I like to keep imports roughly grouped by category.